### DISCUSSION TITLE
v-bottom-sheet in v3

### DISCUSSION THREAD

**Discussion by archnim** [vuetifyjs/vuetify#15831] [Ideas] [answered]
Hello world ! I always expected a better version of v-bottom-sheet, and I hoped it would be implemented in version 3. But I seems like that component has been totally removed in this version. Is it a mistake or is there a reason for that ?

**Comment by 1Luc1** [comment, marked as answer]
> This is essentially `<v-navigation-drawer location="bottom" temporary>`

from KaelWD @ https://github.com/vuetifyjs/vuetify/issues/13466#issuecomment-1219139127
